### ISSUE TITLE
Rocket Chat crashes on Mac OS X with segmentation Fault

### ISSUE THREAD

**Issue by Deepakkothandan** [RocketChat/Rocket.Chat.Electron#192]
- Mac OS X: Mountain Lion
- Rocket Chat Version 1.3.1 : 

- [x] I have tested with the latest application version
- [x] I can simulate the issue easily
## Description

When I open the app, the app opens up and then crashes in less than 5 second even before loading
#### Current Behavior

Please find attached the logs while opening the app

```
 +[NSColor colorWithRed:green:blue:alpha:]: unrecognized selector sent to class 0x7fff7a990cc0
 An uncaught exception was raised
 +[NSColor colorWithRed:green:blue:alpha:]: unrecognized selector sent to class 0x7fff7a990cc0
(
        0   CoreFoundation                      0x00007fff96755b06 __exceptionPreprocess + 198
        1   libobjc.A.dylib                     0x00007fff929263f0 objc_exception_throw + 43
        2   CoreFoundation                      0x00007fff967ec26a +[NSObject(NSObject) doesNotRecognizeSelector:] + 186
        3   CoreFoundation                      0x00007fff9674402e ___forwarding___ + 414
        4   CoreFoundation                      0x00007fff96743e18 _CF_forwarding_prep_0 + 232
        5   Electron Framework                  0x000000010249990f _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 2543
        6   Electron Framework                  0x000000010249999b _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 2683
        7   Electron Framework                  0x0000000102499820 _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 2304
        8   Electron Framework                  0x00000001024996aa _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 1930
        9   Electron Framework                  0x0000000102499299 _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 889
        10  Electron Framework                  0x0000000102499aa1 _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 2945
        11  Electron Framework                  0x000000010249a6e1 _ZN4atom13TrayIconCocoa8SetTitleERKNSt3__112basic_stringIcNS1_11char_traitsIcEENS1_9allocatorIcEEEE + 49
        12  Electron Framework                  0x000000010243acee _ZN4atom3api4Tray8SetTitleEPN4mate9ArgumentsERKNSt3__112basic_stringIcNS5_11char_traitsIcEENS5_9allocatorIcEEEE + 30
        13  Electron Framework                  0x000000010243d975 _ZN4base8internal15RunnableAdapterIMN4atom3api4TrayEFvPN4mate9ArgumentsERKNSt3__112basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEEEE3RunEPS4_RKS7_SG_ + 69
        14  Electron Framework                  0x000000010243d91b _ZN4base8internal12InvokeHelperILb0EvNS0_15RunnableAdapterIMN4atom3api4TrayEFvPN4mate9ArgumentsERKNSt3__112basic_stringIcNS9_11char_traitsIcEENS9_9allocatorIcEEEEEEENS0_8TypeListIJRKPS5_RKS8_SH_EEEE8MakeItSoESK_SO_SQ_SH_ + 75
        15  Electron Framework                  0x000000010243d8be _ZN4base8internal7InvokerINS_13IndexSequenceIJEEENS0_9BindStateINS0_15RunnableAdapterIMN4atom3api4TrayEFvPN4mate9ArgumentsERKNSt3__112basic_stringIcNSC_11char_traitsIcEENSC_9allocatorIcEEEEEEEFvPS8_SB_SK_ENS0_8TypeListIJEEEEESR_NS0_12InvokeHelperILb0EvSN_NSQ_IJRKSO_RKSB_SK_EEEEEFvSV_SX_SK_EE3RunEPNS0_13BindStateBaseESV_SX_SK_ + 78
        16  Electron Framework                  0x000000010243d774 _ZNK4base8CallbackIFvPN4atom3api4TrayEPN4mate9ArgumentsERKNSt3__112basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEEEE3RunERKS4_RKS7_SG_ + 84
        17  Electron Framework                  0x000000010243d66a _ZN4mate8internal7InvokerINS0_13IndicesHolderIJLm0ELm1ELm2EEEEJPN4atom3api4TrayEPNS_9ArgumentsERKNSt3__112basic_stringIcNSA_11char_traitsIcEENSA_9allocatorIcEEEEEE18DispatchToCallbackEN4base8CallbackIFvS7_S9_SI_EEE + 26
        18  Electron Framework                  0x000000010243d525 _ZN4mate8internal10DispatcherIFvPN4atom3api4TrayEPNS_9ArgumentsERKNSt3__112basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEEEE18DispatchToCallbackERKN2v820FunctionCallbackInfoINSJ_5ValueEEE + 181
        19  libnode.dylib                       0x00000001068b29f3 _ZN2v88internal25FunctionCallbackArguments4CallEPFvRKNS_20FunctionCallbackInfoINS_5ValueEEEE + 163
        20  libnode.dylib                       0x00000001068dbf3c _ZN2v88internal8Builtins17InvokeApiFunctionENS0_6HandleINS0_10JSFunctionEEENS2_INS0_6ObjectEEEiPS6_ + 1644
        21  libnode.dylib                       0x00000001068e0001 _ZN2v88internal8BuiltinsD1Ev + 16177
        22  ???                                 0x0000277362508b3b 0x0 + 43376524167995
        23  ???                                 0x0000277362536763 0x0 + 43376524355427
    )
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '+[NSColor colorWithRed:green:blue:alpha:]: unrecognized selector sent to class 0x7fff7a990cc0'
    *** First throw call stack:
    (
        0   CoreFoundation                      0x00007fff96755b06 __exceptionPreprocess + 198
        1   libobjc.A.dylib                     0x00007fff929263f0 objc_exception_throw + 43
        2   CoreFoundation                      0x00007fff967ec26a +[NSObject(NSObject) doesNotRecognizeSelector:] + 186
        3   CoreFoundation                      0x00007fff9674402e ___forwarding___ + 414
        4   CoreFoundation                      0x00007fff96743e18 _CF_forwarding_prep_0 + 232
        5   Electron Framework                  0x000000010249990f _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 2543
        6   Electron Framework                  0x000000010249999b _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 2683
        7   Electron Framework                  0x0000000102499820 _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 2304
        8   Electron Framework                  0x00000001024996aa _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 1930
        9   Electron Framework                  0x0000000102499299 _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 889
        10  Electron Framework                  0x0000000102499aa1 _ZN4base7WeakPtrINS_16ObserverListBaseIN4atom16TrayIconObserverEEEED2Ev + 2945
        11  Electron Framework                  0x000000010249a6e1 _ZN4atom13TrayIconCocoa8SetTitleERKNSt3__112basic_stringIcNS1_11char_traitsIcEENS1_9allocatorIcEEEE + 49
        12  Electron Framework                  0x000000010243acee _ZN4atom3api4Tray8SetTitleEPN4mate9ArgumentsERKNSt3__112basic_stringIcNS5_11char_traitsIcEENS5_9allocatorIcEEEE + 30
        13  Electron Framework                  0x000000010243d975 _ZN4base8internal15RunnableAdapterIMN4atom3api4TrayEFvPN4mate9ArgumentsERKNSt3__112basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEEEE3RunEPS4_RKS7_SG_ + 69
        14  Electron Framework                  0x000000010243d91b _ZN4base8internal12InvokeHelperILb0EvNS0_15RunnableAdapterIMN4atom3api4TrayEFvPN4mate9ArgumentsERKNSt3__112basic_stringIcNS9_11char_traitsIcEENS9_9allocatorIcEEEEEEENS0_8TypeListIJRKPS5_RKS8_SH_EEEE8MakeItSoESK_SO_SQ_SH_ + 75
        15  Electron Framework                  0x000000010243d8be _ZN4base8internal7InvokerINS_13IndexSequenceIJEEENS0_9BindStateINS0_15RunnableAdapterIMN4atom3api4TrayEFvPN4mate9ArgumentsERKNSt3__112basic_stringIcNSC_11char_traitsIcEENSC_9allocatorIcEEEEEEEFvPS8_SB_SK_ENS0_8TypeListIJEEEEESR_NS0_12InvokeHelperILb0EvSN_NSQ_IJRKSO_RKSB_SK_EEEEEFvSV_SX_SK_EE3RunEPNS0_13BindStateBaseESV_SX_SK_ + 78
        16  Electron Framework                  0x000000010243d774 _ZNK4base8CallbackIFvPN4atom3api4TrayEPN4mate9ArgumentsERKNSt3__112basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEEEE3RunERKS4_RKS7_SG_ + 84
        17  Electron Framework                  0x000000010243d66a _ZN4mate8internal7InvokerINS0_13IndicesHolderIJLm0ELm1ELm2EEEEJPN4atom3api4TrayEPNS_9ArgumentsERKNSt3__112basic_stringIcNSA_11char_traitsIcEENSA_9allocatorIcEEEEEE18DispatchToCallbackEN4base8CallbackIFvS7_S9_SI_EEE + 26
        18  Electron Framework                  0x000000010243d525 _ZN4mate8internal10DispatcherIFvPN4atom3api4TrayEPNS_9ArgumentsERKNSt3__112basic_stringIcNS8_11char_traitsIcEENS8_9allocatorIcEEEEEE18DispatchToCallbackERKN2v820FunctionCallbackInfoINSJ_5ValueEEE + 181
        19  libnode.dylib                       0x00000001068b29f3 _ZN2v88internal25FunctionCallbackArguments4CallEPFvRKNS_20FunctionCallbackInfoINS_5ValueEEEE + 163
        20  libnode.dylib                       0x00000001068dbf3c _ZN2v88internal8Builtins17InvokeApiFunctionENS0_6HandleINS0_10JSFunctionEEENS2_INS0_6ObjectEEEiPS6_ + 1644
        21  libnode.dylib                       0x00000001068e0001 _ZN2v88internal8BuiltinsD1Ev + 16177
        22  ???                                 0x0000277362508b3b 0x0 + 43376524167995
        23  ???                                 0x0000277362536763 0x0 + 43376524355427
    )
 com.apple.launchd[1] (com.apple.xpcd.F5010000-0000-0000-0000-000000000000[9282]): Exited: Killed: 9
kernel[0]: memorystatus_thread: idle exiting pid 9282 [xpcd]
com.apple.launchd.peruser.501[123] (com.apple.pbs[9284]): Exited: Killed: 9
 kernel[0]: memorystatus_thread: idle exiting pid 9284 [pbs]
om.apple.launchd.peruser.501[123] (com.apple.ReportCrash[9293]): Job appears to have crashed: Segmentation fault: 11
local ReportCrash[9295]: Saved crash report for ReportCrash[9293] version 418 to Library/Logs/DiagnosticReports/ReportCrash_2016-05-17--MacBook-Air.crash
Removing excessive log: Library/Logs/DiagnosticReports/ReportCrash_2016-04-28--MacBook-Air.crash
local ReportCrash[9296]: DebugSymbols was unable to start a spotlight query: spotlight is not responding or disabled.
com.apple.launchd[1]: catch_mach_exception_raise_state_identity(): PID: 9296 thread: 0x2bc9b type: 0xa code: 0x7fff4fcef574 codeCnt: 0x2 flavor: 0x7fff4fcef584 old_state: 0x7fff4fcef58c old_stateCnt: 0x2c new_state: 0x7fff4fceed2c new_stateCnt: 0x7fff4fceed28
com.apple.launchd.peruser.501[123] ([0x0-0x492492].chat.rocket[9285]): Job appears to have crashed: Abort trap: 6
 com.apple.launchd[1] (com.apple.ReportCrash.Root[9296]): Job appears to have crashed: Segmentation fault: 11
 com.apple.launchd[1] (com.apple.coresymbolicationd[9294]): Exited: Killed: 9
local ReportCrash[9295]: Client side error: Connection interrupted
kernel[0]: memorystatus_thread: idle exiting pid 9294 [coresymbolicatio]
 com.apple.launchd.peruser.501[123] (com.apple.tccd[9290]): Exited: Killed: 9
kernel[0]: memorystatus_thread: idle exiting pid 9290 [tccd]
 com.apple.launchd.peruser.501[123] (com.apple.CalendarAgent[9288]): Exited: Killed: 9
kernel[0]: memorystatus_thread: idle exiting pid 9288 [CalendarAgent]
com.apple.launchd[1] (com.apple.cfprefsd.xpc.daemon[9283]): Exited: Killed: 9
kernel[0]: memorystatus_thread: idle exiting pid 9283 [cfprefsd]
M.local Finder[178]: Unable to stat pipe path /Library/Application Support/Google/Drive/GoogleDriveIpcPipe
local Finder[178]: Unable to get underlying pipe path.
com.apple.launchd.peruser.501[123] (com.apple.cfprefsd.xpc.agent[9278]): Exited: Killed: 9
kernel[0]: memorystatus_thread: idle exiting pid 9278 [cfprefsd]
```
#### Expected Behavior

The app opens and stays open
##

**Comment by Deepakkothandan** [comment]
This problem occurred no more, after upgrading to OS X El Capitan

**Comment by engelgabriel** [comment]
Fixed on version 2.2.0
